Jenkinsfile

```diff
@@ -445,6 +445,7 @@ try {
           timeout(time: max_time, unit: 'MINUTES') {
             init_git()
             docker_run('armv7', 'build_armv7', false)
+            archiveArtifacts artifacts: 'build/*.whl,build/libmxnet.*'
```

Comment:
I'd like to prevent the usage of archive artifacts as it is going to increase the permanently stored size of every run by a LOT - and it won't be possible to distinguish the different wheels from each other.

Reply:
we need to publish these wheels, what alternative you propose. Shall we push them to S3?

Reply:
No, this is CI and not CD, so we're not supposed (and not allowed) to publish anything. Please remove it entirely.